loadbalancer.cfhighlander.rb

```diff
@@ -17,4 +17,4 @@
     subnet_parameters({'compute'=>{'name'=>'Compute'}}, maximum_availability_zones) if defined?(loadbalancer_scheme) && loadbalancer_scheme == 'internal'
```

Comment:
@Guslington Subnet parameter now can also be defined as `ComponentParam` rather than using helper functions from vpc component. 
Look at https://github.com/theonestack/hl-component-ecs-service/blob/master/ecs-services.highlander.rb#L17 for reference. In addition to this example, we can also add `type: 'AWS::EC2::Subnet::Id'`  as named parameter to `ComponentParam` dsl statement.